wzdx/raw_to_standard/icone.py

```diff
@@ -0,0 +1,226 @@
+from collections import OrderedDict
+import json
+import time
+from typing import Union
```

Comment:
Union doesn't seem to be used, can we remove?

Reply:
done